File: Aplicaciones Distribuidas/PARCIAL3/PATRONES DE DISENO VIDEO 3/APLICATIVO/Java/TuberiaYFiltrosGrupo05/src/ec/edu/monster/tuberiayfiltrosgrupo05/filters/ConvertToCharArrayHandler.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.tuberiayfiltrosgrupo05.filters;

/**
 *
 * @author santi
 */
public class ConvertToCharArrayHandler implements Handler<String, char[]> {

    @Override
    public char[] process(String input) {
        char[] charArray = new char[input.length()];
        for (int i = 0; i < input.length(); i++) {
            charArray[i] = input.charAt(i);
        }
        return charArray;
    }

}

File: Aplicaciones Distribuidas/PARCIAL3/PATRONES DE DISENO VIDEO 3/APLICATIVO/Dotnet/TuberiaYFiltrosGrupo05/TuberiaYFiltrosGrupo05/filters/ConvertToCharArrayHandler.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TuberiaYFiltrosGrupo05.filters
{
    public class ConvertToCharArrayHandler : IHandler<string, char[]>
    {
        public char[] Process(string input)
        {
            char[] charArray = new char[input.Length];
            for (int i = 0; i < input.Length; i++)
            {
                charArray[i] = input[i];
            }
            return charArray;
        }
    }

}
